default/views.py

```diff
@@ -210,15 +210,20 @@ def perform_to_json(request):
 
     output_name = output_dir.path + '.json'
     extension = os.path.splitext(input_file.path)[1]
-    if extension == '.zip':
-        input_file_path = output_dir.path + '/tmp'
```

Comment:
Why `/tmp` dissapears in the new code?

Reply:
We can only use `output_dir.path` to create the directory where we extract the files, and in the end we remove it with `shutil.rmtree (input_file_path)`, but I can add '/tmp'.

Reply:
Ok, I understand.